src/UXCapture.js

```diff
@@ -47,8 +62,11 @@ const UXCapture = {
 		_view.update(zoneConfigs);
 	},
 
-	// TODO: SPA support in subsequent ticket
-	startTransition: () => {},
+	startTransition: () => {
+		UXCapture._clearMarksAndMeasures();
+
+		_view.startTransition();
```

Comment:
We should put some safety checks around this in case `_view` is undefined or an instance of `PageView`.

```
if (_view && _view.startTransition) {
    _view.startTransition();
}
```

Reply:
I'll write a test that spells out calling UXCapture.startTransition() before UXCapture.startView() and do some exception throwing ;)

Reply:
Well, it's the other way around now so not doing this.